flumeserver/api/flumeAPI_test.go

```diff
@@ -110,14 +110,17 @@ var (
 	genericAddr   = "0x3cd751e6b0078be393132286c442345e5dc49699"
 )
 
-func TestFumeAPI(t *testing.T) {
+func TestFlumeAPI(t *testing.T) {
 	db, err := connectToDatabase()
 	if err != nil {
 		t.Fatal(err.Error())
 	}
 	defer db.Close()
 	f := NewFlumeAPI(db, 1)
 
+	offset := new(int)
+	*offset = 0
```

Comment:
already 0

Reply:
Also, I think most of the places that use this could just use `nil`, since the functions you're calling with it will check for nil values.